etc/kayobe/inventory/group_vars/all/firewall

```diff
@@ -0,0 +1,369 @@
+---
+#########################################
+# Stackhpc default firewall configuration
+#########################################
+
+stackhpc_firewalld_zones: |
+  {% set network_zones = [] %}
+  {% for network in network_interfaces %}
+  {% if network | net_zone is not none %}
+  {% set _ = network_zones.append({'zone': network | net_zone }) %}
+  {% endif %}
+  {% endfor %}
+  {{ network_zones }}
+
+stackhpc_firewalld_rules: |
+  {{  (stackhpc_controller_firewalld_rules if 'controllers' in group_names else []) +
+      (stackhpc_compute_firewalld_rules if 'compute' in group_names else []) +
+      (stackhpc_storage_firewalld_rules if 'storage' in group_names else []) +
+      (stackhpc_monitoring_firewalld_rules if 'monitoring' in group_names else []) +
+      (stackhpc_seed_firewalld_rules if 'seed' in group_names else []) +
+      (stackhpc_seed_hypervisor_firewalld_rules if 'seed_hypervisor' in group_names else []) +
+      (stackhpc_wazuh_manager_infra_vm_firewalld_rules if 'wazuh_manager' in group_names else []) +
+      (stackhpc_ansible_control_infra_vm_firewalld_rules if 'ansible_control' in group_names else []) }}
+
+###############################################################################
+# Controller firewalld rules
+
+stackhpc_controller_firewalld_rules: |
```

Comment:
I think you need to remove newlines:
```suggestion
stackhpc_controller_firewalld_rules: >-
```

Reply:
Both appear to work. I've gone with folded because it makes more logical sense to me

Reply:
Actually `|` is literal, and `>` is folded. `-` is the chomping mode...

Reply:
Ah, I must've reverted the change at some point. I did switch it but it appears to have switched back.